frontend/taipy/src/ScenarioViewer.tsx

```diff
@@ -108,38 +116,52 @@ const tagsAutocompleteSx = {
     maxWidth: "none",
 };
 
+type SequenceFull = [string, string, string[], boolean, boolean];
+// enum SeFProps {
```

Comment:
Do you really want to keep these?

Reply:
yes, I think it's important to describe somewhere the format